web/src/components/merchant/sign-up/SignUpForm.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import FormRow from 'components/common/FormRow';
+import useSignUpForm from 'components/merchant/sign-up/hooks/useSignUpForm';
+import Button from 'muicss/lib/react/button';
+import Form from 'muicss/lib/react/form';
+import styled from 'styled-components';
+
+const StyledForm = styled(Form)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const StyledButton = styled(Button)`
+  height: 40px;
+  width: 200px;
+  border-radius: 20px;
+  border: none;
+
+  background: var(--dark-gray);
+  color: white;
+  font-size: 18px;
+  font-weight: bolder;
+  text-transform: uppercase;
+  margin-top: 20px;
+`;
+
+const SignUpForm = () => {
+  const {disabled, errors, onSubmit, validations} = useSignUpForm();
+  return (
+    <StyledForm>
+      <FormRow
+        label="Name"
+        inputType="text"
+        forwardedRef={validations.name}
+        error={errors?.name?.message}
```

Comment:
Why does `errors` need optional chaining here? If I'm not wrong I think only the fields are possibly undefined? Same for the rest of the form rows!
```suggestion
        error={errors.name?.message}
```

Reply:
ohh. I thought that errors might be undefined also. But I think you're right. Only the fields are possibly undefined.